.github/workflows/create-pr-to-update-release-version.yml

```diff
@@ -0,0 +1,108 @@
+name: PR to update homebrew and minikube
+
+# This action will run after the "Publish container" is succesfully completed
+# https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run
+on:
+  workflow_run:
+    workflows: ["Publish container"]
+    types:
+      - completed
+env:
+  LATEST_HEADLAMP_TAG: latest
+jobs:
+  create_pr_to_upgrade_homebrew_minikube:
+    name: Create PR to upgrade homebrew and minikube
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout headlamp repo 
+        uses: actions/checkout@v2 
+        with:
+          path: headlamp
+          fetch-depth: 0
+      - name: Configure Git 
+        run: | 
+          git config --global user.name 'yolossn'
+          git config --global user.email 'sannagaraj@microsoft.com'
+      - name: Get headlamp latest tag 
+        run: |
+          cd headlamp 
+          latestTag=$(git tag --list --sort=version:refname 'v*' | tail -1)
+          echo "LATEST_HEADLAMP_TAG=$latestTag" >> $GITHUB_ENV
+          echo $latestTag
+      - name: Sync homebrew-cask fork from upstream
+        run: |
+          gh repo sync kinvolk/homebrew-cask
+        env:
+          GITHUB_TOKEN: ${{ secrets.TOKEN }}
```

Comment:
This could also be used as the token of the user who's triggered the request AFAIU. (I'm not entirely sure this works for tag-triggered actions but maybe it's worth a shot)
https://docs.github.com/en/actions/security-guides/automatic-token-authentication

Reply:
The token that is automatically added doesn't work because it has the scope particular to this repository alone and we need a token which access other repositories too. Using the token that is automatically added returns an unauthorised error.

Reply:
Okay. Then let's call the secret KINVOLK_REPOS_TOKEN or something more particular, since we have already many secrets.

Reply:
Done 👍🏽